mwsql-blogpost.ipynb

```diff
@@ -0,0 +1,557 @@
+{
```

Comment:
<p>"Note that we haven't had to worry about where to download the files from, nor whether they're compressed (.gz)."</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/blancadesal/wm-tutorials-and-blogposts/pull/3/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
<p>Done</p>